website/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to Dashboards
+
+Contributions are very welcome! Just fork the repository, develop in a branch and submit a pull request.
+
+## Commit Messages
+
+We follow [conventional commits](https://www.conventionalcommits.org) when writing commit messages.
+The messages themselves should help future developers understand **why** changes were made.
+
+## Code Style
+
+We value clean code, here are some guidelines on what we consider clean code:
+
+-   Use expressive names (variables, classes, function), avoid abbreviations unless they are really common sense
+-   Small functions and classes
+-   Reasonable test coverage: We don't require 100% test coverage, but we do require tests for all non-trivial code.
+    Every piece of the code should at least be tested in the happy path.
+-   Develop code with testability in mind.
+
+### Frontend
+
+We make use of two frameworks: Astro, React, and Tailwindcss.
+
+-   use Astro when possible (for performance reasons)
+    -   however avoid script tags, since they are harder to maintain
+-   when possible use css over js for interactivity
```

Comment:
I think we could add why:
* it often yields less and simpler code
* we can profit from browser optimizations (modern browsers are highly optimized for parsing HTML and CSS - when using JS that's not immediately possible)
* a lot of things (modals, overlays, etc) can be done with CSS only using standardized solutions (and add a reference to DaisyUI because it has many example how to do it?)